about/translations/translations.js

```diff
@@ -69,16 +69,23 @@ $(document).ready(function(){
 
         var total = translated + fuzzy + untranslated;
 
-        var pct_translated   = (total == 0 ? 0 : 100 * (translated / total));
-        var pct_fuzzy        = (total == 0 ? 0 : 100 * (fuzzy / total));
+        round = function(number, precision) {
```

Comment:
Is the absence of `var` intentional here? Sorry for my JS-ignorance...

Reply:
Nope, I can add that, good spot.